frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/index.tsx

```diff
@@ -77,14 +49,83 @@ class SubCategories extends React.Component<
         this.setState({ isLoading: false });
       });
   }
+
+  handleDelete = (id: number) => {
+    this.setState({ isLoading: true });
+    axios
+      .delete(
+        window.location.origin + `/core/academix/admin/sub-categories/${id}`
+      )
+      .then((result: AxiosResponse<Category[]>) => {
```

Comment:
This won't return a category array. It will return a category

Reply:
yes my bad